lucene/directory-provider/src/test/java/org/infinispan/hibernate/search/TwoNodesTest.java

```diff
@@ -60,6 +60,7 @@ public void testSomething() {
       } finally {
          nodeC.close();
       }
+      waitMembersCount(nodea, SimpleEmail.class, 2);
```

Comment:
@Sanne people starting real nodes on real machines can already use `configuration.transport.initialClusterSize`. But the test kills a node and for some reason needs to check that the view is updated on the other nodes -- I don't see why users would do this.